msg/sensor_accel.msg

```diff
@@ -8,6 +8,9 @@ float32 y                 # acceleration in the FRD board frame Y-axis in m/s^2
 float32 z                 # acceleration in the FRD board frame Z-axis in m/s^2
 
 float32 temperature       # temperature in degrees Celsius
+float32 scale             # sensor resolution scale (LSB/g)
+
+uint16 dynamic_range      # dynamic range of the sensor (g)
```

Comment:
Maybe it's slightly wasteful, but I would just include this as a float32 in the same units the data (m/s^2).

Reply:
Sounds great!